docs/getting_started/prepare_environment.md

```diff
@@ -0,0 +1,69 @@
+# Prerequisites: Prepare your Environment
+
```

Comment:
Do we want to add a toolkit overview to this file?

Reply:
We have a set of docs explaining our toolkit here: https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/docs/how_it_works/0_intro.md

It is quite in-depth though, so maybe just phrase it as a reference/fyi type thing.